modules/default-flatpaks/system-flatpak-setup

```diff
@@ -51,6 +51,14 @@ FLATPAK_LIST=$(flatpak list --system --columns=application)
 INSTALL_LIST_FILE="/etc/flatpak/system/install"
 REMOVE_LIST_FILE="/etc/flatpak/system/remove"
 
+function notify-send-pre-install {
+     user_name=$(loginctl list-sessions --output=json | jq -r '.[].user')
+     uid=$(loginctl list-sessions --output=json | jq -r '.[].uid')
+     xdg_runtime_path="/run/user/$uid"
+     display_var=$(printenv DISPLAY)
+     sudo -u "$user_name" DBUS_SESSION_BUS_ADDRESS=unix:path="$xdg_runtime_path"/bus DISPLAY="$display_var" notify-send "Flatpak Installer" "Started install of some system flatpaks" --app-name="Flatpak Installer" -u NORMAL
+}
```

Comment:
Should replace/remove the word "some" from this messages. An alternative could be "default".

Reply:
I set "some" in both system & user flatpak setup.

I will have to think on which word is more suitable for both, since "default" depends on if the maintainer uses system or user flatpaks as their "defaults".

If I omit "some" word completely, it can be confusing to the user, thinking all of his flatpaks will be removed, instead of "some".

Reply:
Some is fine for uninstall IMO, for install it feels weird.

Reply:
> Some is fine for uninstall IMO, for install it feels weird.

I use "listed" term now for both. It's more accurate & prevents the confusion to the user.

Reply:
> Are the flatpaks listed in the notif? If not, a non-maintainer user might get even more confused.

No, they're not.

Do you have any ideas on how to solve this? This task became harder then doing code itself lol

Reply:
Updated